app/jobs/resolution_proofing_job.rb

```diff
@@ -25,6 +25,11 @@ def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
 
     applicant_pii = decrypted_args[:applicant_pii]
 
+    if use_lexisnexis_ddp_threatmetrix_before_rdp_instant_verify?
+      proofer_result = proof_lexisnexis_ddp_with_threatmetrix(applicant_pii)
+      return proofer_result if proofer_result.failed?
```

Comment:
this will return for network failures only right? (we still want to continue with the proofing process if we successfully hear back from LN but get a negative result)

Reply:
ah yes.  we spoke about this today.  i'll change it.